### PR TITLE
feat: add transponder/squawk related documentation for dcs

### PR DESCRIPTION
**Pull request by M3ales**
## Summary

Aims to add consolidated page of instructions on each 'popular' airframe, on the use of the mode 3 transponder and IDENT.

I added a section explaining what can be seen from the LotAtc/Controller side.

Related [thread on Discord](https://discord.com/channels/1099799536416862330/1102555371404525640)

## Airframes to include initially

- [x] F/A 18 @M3ales 
- [x] F1 (`@Generic#2998`)
- [ ] ~~F16~~
- [ ] ~~F14 [Rio Only, otherwise SRS] `Merlin said he might look at it` ?~~
- [ ] ~~JF17~~
- [ ] ~~SRS Overlay Aircraft [SRS] -- Harrier, F15, A10C, etc.~~

*Will be added in subsequent prs*

## Media Size/Formats

800px webp for both animated and not animated items.

## Presentation

~~The F/A 18 is presented as a 'walkthrough' of sorts, with just steps and prose to match.~~

~~The F1 is presented as a checklist. Both are interesting but we haven't decided which to work with.~~

~~@QuantifyGG prefers the F/A 18 style.~~

Content tabs added to allow for both.

### CODE REVIEW

File: docs/general-knowledge/index.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # General Knowledge
 
 The general knowledge section contains the following information:
+
 - [Radiotelephony](./radiotelephony.md)
 - [Altimetry](./altimetry.md)
 - [Meteorology](./meteorology.md)
 - [Flight Rules](./flight_rules.md)
+- [Squawk](./dcs_mode_3_squawk.md)
```
Comment (M3ales) [inline docs/general-knowledge/index.md]: Not really sure where to add this, or how to categorise it. Most likely `Ground`, but have just added it to the bottom here for dev in the meantime.

**Comment by evwltrs** [comment]
@M3ales What planes are remaining for this to be complete?

**Comment by evwltrs** [comment]
I'd like to suggest using animated webp images instead of GIF's as they are much more bandwidth efficient.

**Comment by M3ales** [comment]
I only have f18 and F1, there's a tentative list of what's needed on the discord which I'll move here.

**Comment by M3ales** [comment]
@evwltrs updated pr with relevant information about it. Could look at webp, just easy for me to convert what I have to gifs, decently well optimised also.

**Comment by M3ales** [comment]
Looked around a bit, `avif` might be nice to go with bandwidth wise. Got a 3mb file down to 700kb in the conversion.

Biggest blank space here is its not supported by edge, or android builtin browsers.

Could do webp fine though -- just a bit larger.

**Comment by evwltrs** [comment]
> Looked around a bit, `avif` might be nice to go with bandwidth wise. Got a 3mb file down to 700kb in the conversion.
> 
> Biggest blank space here is its not supported by edge, or android builtin browsers.
> 
> Could do webp fine though -- just a bit larger.

I reckon webp as it's fully supported even if it's a bit bigger. Just want to avoid gif because if we have 10 of these images and they are 2mb each that will end up being a 20mb load for the page which will end up taking a significant amount of time for people on slower connections. Hopefully webp should allow this to be a bit lower, or I may look into lazy loading for the docs.

**Comment by M3ales** [comment]
> > Looked around a bit, `avif` might be nice to go with bandwidth wise. Got a 3mb file down to 700kb in the conversion.
> > Biggest blank space here is its not supported by edge, or android builtin browsers.
> > Could do webp fine though -- just a bit larger.
> 
> I reckon webp as it's fully supported even if it's a bit bigger. Just want to avoid gif because if we have 10 of these images and they are 2mb each that will end up being a 20mb load for the page which will end up taking a significant amount of time for people on slower connections. Hopefully webp should allow this to be a bit lower, or I may look into lazy loading for the docs.

Done some of it, webp/avif. Will convert rest to webp. Recommend we squash merge this branch when we do to prevent a stack of binary garbage ending up in the git db.


It'd also be valuable to add the requirements for filetypes somewhere, maybe readme. And enforce it/modify existing files too. Don't mind submitting prs for the existing images while I'm doing the convert spam.

**Comment by evwltrs** [comment]
@QuantifyGG What do you think about enforcing specific file types for images and videos?

**Comment by QuantifyGG** [comment]
As long as it's kept consistent, I see no problem with enforcing specific file types. It's probably for the best actually, webp is a good format!

**Comment by M3ales** [comment]
@evwltrs converted my stuff over to webp.

@QuantifyGG don't have access to make docs for other aircraft listed, just own a `Hornet`. I'm also away from gaming setup for the rest of this week. It should be fine to merge just these two aircraft, and I can make separate prs for other aircraft or others can do that as needed.

Need to add:
- F16
- F14
- JF17
- SRS Overlay Example

File: README.md [hunk 2]
```diff
@@ -26,3 +26,8 @@ Ensure you have docker installed. Run the compose without detatch to get tty (te
 ```
 docker-compose up
 ```
+
+
+## Images and Video
+
+Images and video are large in older formats (`jpg/png/gif`). Payload size was raised as an issue, and it was decided that to reduce delays for FCP on older devices: All images submitted as part of PRs must be converted to `webp`. This includes still images and animated `gifs`, `apngs` etc. Video may be added as `webp` or `webm` if audio is required. `avif` was considered but was skipped due to it's adoption rate.
```
Comment (QuantifyGG) [inline README.md]: Typo in "it's" but otherwise fine

**Review by QuantifyGG — APPROVED** [review]
Looks good, thanks! We can open PRs for individual airplanes in the future

**Comment by M3ales** [comment]
Also going to add here: try out the tabbed thing I mentioned to you on discord. Just added it. Theme change for content linking, and a plugin add for tabbed.

**Pull request merged**